src/Http/Controllers/ArticleController.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
Can we pluralize the filename to be inline with the naming convention.
`src/Http/Controllers/ArticleController.php` -> `src/Http/Controllers/ArticlesController.php`